.NET/Microsoft.Recognizers.Text.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2026
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30204.135
```

Comment:
Be careful not to change the solution unless intentional.

Reply:
This was highly unintentional. I must check my settings because I had to keep manually revert this change. I forgot it this time.